esp32-wroom-rp/src/protocol.rs

```diff
@@ -55,6 +65,7 @@ pub(crate) struct NinaNoParams {
 }
 
 impl NinaConcreteParam for NinaNoParams {
```

Comment:
I believe we can no remove `NinaNoParams` and all other code related to that struct.